app/component/itinerary/ItineraryDetails.js

```diff
@@ -290,6 +293,18 @@ class ItineraryDetails extends React.Component {
                   legs={itinerary.legs}
                 />
               )),
+            this.props.navigate && false && (
```

Comment:
I this navigate properties should be read at config, like config.navigate? In config file that properties is named like this: navigation. So I think we need use it here also (config.navigation).